docs/admin-guide/load-config.md

```diff
@@ -148,6 +115,9 @@ By giving a path to each individual vcf2cytosure-file these are made available
 for download on the case page. Such SV files can be visualized using standard arrayCGH
 analysis tools. See [vcf2cytosure](https://github.com/NBISweden/vcf2cytosure/blob/master/README.md).
 
-[peddy]: https://github.com/brentp/peddy
-[multiqc]: https://github.com/ewels/multiqc
 [arriba]: https://arriba.readthedocs.io/en/latest/
+[bjhall]: https://github.com/bjhall/upd
+[multiqc]: https://github.com/ewels/multiqc
+[peddy]: https://github.com/brentp/peddy
+[tiddit]: https://github.com/SciLifeLab/TIDDIT
+[rhocall]: https://github.com/dnil/rhocall
```

Comment:
r of rhocall comes before t of tiddit, in case you wanted then in alphabetical order.